source/app/models/logic.rb

```diff
@@ -0,0 +1,41 @@
+class Logic
+  attr_reader :status
+  def initialize(game, choice1, choice2, player1, player2)
+    @game = game
+    @status = true
+    @choice1, @choice2 = choice1, choice2
+    @player1, @player2 = player1, player2
+    checker
+    if @status
+      rules
+      choose_winner
+    end
+  end
+
+  def checker
+    @status = false if @choice1 == @choice2
+  end
+  #set the standard of what beats what
+  def rules
```

Comment:
The logic here and in `choose_winner` is hard to follow. I'd use a hash to store the relations:

    BEATS = {
        'charmander' => 'squirtle',
        'squirtle' => 'bulbasaur',
        'bulbasaur' => 'charmander',
    }

And then check it in your access methods,

    def winner
      return nil if @choice1 == @choice2
      if BEATS[@choice1] == @choice2
        @player1
      else
        @player2
      end
    end

    def loser
      return nil unless winner
      return @player2 if winner == @player1
      @player1
   end

And so on.